docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -100,16 +109,41 @@ Specific kinds of architecture guides include:
 
 
 ## General style rules
+
 Please refer to this style guide when determining how address questions about
 English grammar, usage, and so on. Since many of these rules have equally
 justifiable alternatives, a style guide prevents debates over these questions
 from getting in the way of documenting Teleport.
 
+### Voice
+
+The documentation should be technically precise and directed toward a technical
+audience, e.g., application developers, site reliability engineers, and security
+engineering team leads. 
```

Comment:
This would preclude the "Connect Your Client" section, which is written for end users of services protected by Teleport.

I suggest that we should be more general here, and say that it should be written for the target audience of the content, and that _most_ of our content is for those reader types.

Reply:
Added a paragraph to the end of the section.